charts/calhoun/values.yaml

```diff
@@ -0,0 +1,168 @@
+# Global values used by this chart
+global:
+  # global.applicationVersion -- What version of the application to deploy
+  applicationVersion: latest
+  # global.terraEnv -- (string) Terget Terra environment name. Required.
+  # @default -- Is set by Helmfile on deploy
+  terraEnv: null
+
+# name -- A name for the deployment that will be substituted into resource definitions
+name: calhoun
+
+imageConfig:
+  # imageConfig.repository -- Image repository
+  repository: gcr.io/broad-dsp-gcr-public/calhoun
+  # imageConfig.tag -- (string) Image tag.
+  # @default -- global.applicationVersion
+  tag: null
+  imagePullPolicy: Always
+
+# replicas -- Number of replicas for the deployment
+replicas: 3
+
+resources:
+  requests:
+    # resources.requests.cpu -- Number of CPU units to request for the deployment
+    cpu: 2
+    # resources.requests.memory -- Memory to request for the deployment
+    memory: 4Gi
+  limits:
+    # resources.limits.cpu -- Number of CPU units to limit the deployment to
+    cpu: 2
+    # resources.limits.memory -- Memory to limit the deployment to
+    memory: 4Gi
+
+probes:
+  readiness:
+    # probes.readiness.enabled -- Whether to configure a readiness probe
+    enabled: true
+    spec:
+      httpGet:
+        path: /status
+        port: 8080
+      timeoutSeconds: 5
+      initialDelaySeconds: 20
+      periodSeconds: 10
+      failureThreshold: 6
+      successThreshold: 1
+  liveness:
+    # probes.liveness.enabled -- Whether to configure a liveness probe
+    enabled: true
+    spec:
+      httpGet:
+        path: /status
+        port: 8080
+      timeoutSeconds: 5
+      initialDelaySeconds: 15
+      periodSeconds: 10
+      failureThreshold: 30 # 5 minutes before restarted
+      successThreshold: 1
+  startup:
+    # probes.startup.enabled -- Whether to configure a startup probe
+    enabled: true
```

Comment:
If calhoun doesn't need to run liquibase migrations on start up, you can set this value  to false. The startup probe exists mainly to ensure k8s won't try to restart a service in the middle of a liquibase migration.